chain/client/src/client_actor.rs

```diff
@@ -743,6 +743,7 @@ impl Handler<Status> for ClientActor {
         } else {
             None
         };
+        let uptime_sec = Clock::utc().timestamp() - self.info_helper.boot_time_seconds;
```

Comment:
Hmm, ideally this would use Instant (which is a monotonic clock) rather than assuming UTC clock is always set correctly on the machine.  Though maybe it’s too nitpicking?  Also not necessarily for this PR.

Reply:
Both timestamps come from the same clock.
As long as the node owner doesn't move the time on the machine by an arbitrary amount, that should be good enough.

Reply:
It’s possible that the clock was set incorrectly when node started and then adjusted.  Again, this is nitpicking.